### DISCUSSION TITLE
Exporting Habor artifacts

### DISCUSSION THREAD

**Discussion by robincher** [goharbor/harbor#15507] [Q&A] [answered]
Hi,

Just a questions if there are any APIs , libraries or ways that i can export my habor artefacts locally?

The current work-around i think of is writing a bash to do docker login and docker pull.

Robin

**Comment by wy65701436** [comment, marked as answer]
In Harbor, it doesn't provide any API to download artifact, you can try with the v2 pull to get the manfest & blob, https://github.com/opencontainers/distribution-spec/blob/main/spec.md#pull.

**Comment by robincher** [comment]
Hi @wy65701436 ,

Thanks we will try with that, appreciate the link.
